packages/amplify-cli/src/index.ts

```diff
@@ -48,11 +50,15 @@ export async function run() {
         throw new Error(verificationResult.message);
       }
     }
-
     rewireDeprecatedCommands(input);
     const context = constructContext(pluginPlatform, input);
+    await attachTelemetry(context);
+    errorHandler = boundErrorHandler.bind(context);
+    process.on('SIGINT', sigIntHandler.bind(context));
```

Comment:
`SIGINT` does not work under Windows, we should follow the docs and use the `readline` module to capture `SIGINT` under Windows: https://nodejs.org/api/readline.html#readline_event_sigint

Reply:
https://github.com/nodejs/node-v0.x-archive/issues/5054

Reply:
I think this has been solved

Reply:
Were you able to test it under Windows that it works?

Reply:
This works on powershell
```js
process.on( "SIGINT", function() {
  console.log( "shutting down" );
  process.exit();
} );

setInterval( () => { }, 2500 );
```